helm-charts/splunk-otel-collector/scripts/lookup-eks-fargate-receiver-node.sh

```diff
@@ -0,0 +1,48 @@
+#! /usr/bin/bash
+set -ex
+
+# If we are the first pod (cluster receiver), set the kubelet stats node filter to only follow labelled nodes.
+# This node label will be set by the second pod.
+if [[ "${K8S_POD_NAME}" == *-0 ]]; then
+  echo "will configure kubelet stats receiver to follow node ${FIRST_CR_REPLICA_NODE_NAME}, as well as use cluster receiver."
+  echo "export CR_KUBELET_STATS_NODE_FILTER='&& labels[\"splunk-otel-is-eks-fargate-cluster-receiver-node\"] == \"true\"'" >/splunk-messages/environ
+  cat /splunk-messages/environ
+
+  # copy config to meet container command args
+  cp /conf/relay.yaml /splunk-messages/config.yaml
+  exit 0
+fi
+
+# Else we are the second pod (wide kubelet stats) label our node to be monitored by the first pod and disable the k8s_cluster receiver.
+# Update our config to not monitor ourselves
+echo "Labelling our fargate node to denote it hosts the cluster receiver"
+
+# download kubectl (verifying checksum)
```

Comment:
Maybe better to use an existing kubectl image instead https://gallery.ecr.aws/bitnami/kubectl ?

Reply:
This is the suggested path from aws and helps ensure we are minimally using their offerings: https://docs.aws.amazon.com/eks/latest/userguide/install-kubectl.html

Reply:
I wasn't able to find an official image w/ kubectl in it and opted not to use a third party for security precautions.

Reply:
This one https://gallery.ecr.aws/bitnami/kubectl is verified and hosted by AWS, seems safe, but it's up to you

Reply:
It's just in the public ecr, w/ a verified bitnami account. If this were for testing I would use it or similar, but not for customer adoption.